harness/determined/exec/harness.py

```diff
@@ -28,9 +28,6 @@ def main(train_entrypoint: str) -> int:
     assert info is not None, "must be run on-cluster"
     assert info.task_type == "TRIAL", f'must be run with task_type="TRIAL", not "{info.task_type}"'
 
-    # TODO: refactor profiling to to not use the cli_cert.
-    certs.cli_cert = certs.default_load(info.master_url)
```

Comment:
I can't tell whether this was easy or hard to refactor. Which is definitely evidence that this is progress. Nice!